RecoLocalTracker/SiPixelRecHits/src/PixelCPEGeneric.cc

```diff
@@ -44,13 +44,16 @@ PixelCPEGeneric::PixelCPEGeneric(edm::ParameterSet const& conf,
   the_eff_charge_cut_highY = conf.getParameter<double>("eff_charge_cut_highY");
   the_size_cutX = conf.getParameter<double>("size_cutX");
   the_size_cutY = conf.getParameter<double>("size_cutY");
+  delta_length_cut = 2.0f;  //seems to be a good value, if needed, will make this externally settable
 
   // Externally settable flags to inflate errors
   inflate_errors = conf.getParameter<bool>("inflate_errors");
   inflate_all_errors_no_trk_angle = conf.getParameter<bool>("inflate_all_errors_no_trk_angle");
 
   NoTemplateErrorsWhenNoTrkAngles_ = conf.getParameter<bool>("NoTemplateErrorsWhenNoTrkAngles");
   IrradiationBiasCorrection_ = conf.getParameter<bool>("IrradiationBiasCorrection");
```

Comment:
Why not also
```suggestion
  noTemplateErrorsWhenNoTrkAngles_ = conf.getParameter<bool>("NoTemplateErrorsWhenNoTrkAngles");
  irradiationBiasCorrection_ = conf.getParameter<bool>("IrradiationBiasCorrection");
```
  etc.